esx_service/cli/vmdkops_admin.py

```diff
@@ -159,8 +159,9 @@ def commands():
                     'func': policy_rm,
                     'help': 'Remove a storage policy',
                     'args': {
-                        'name': {
-                            'help': 'Policy name'
+                        '--name': {
```

Comment:
sorry I am late. Can you make sure admin_cli information is in sync with the proposed changes?

e.g. https://github.com/vmware/docker-volume-vsphere/blob/master/docs/user-guide/admin-cli.md#remove-3